cmd/immuclient/immuc/getcommands.go

```diff
@@ -31,6 +31,9 @@ func (i *immuc) GetTxByID(args []string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf(" \"%v\" is not a valid id number", args[0])
 	}
+	if id == 0 {
+		return "", fmt.Errorf(" invalid tx id (should be bigger than 0)")
```

Comment:
First white space may not be needed

Reply:
Updated the message. I think equality is fine, we have an uint64.